### PR TITLE
Italian language implementation

### PR DESCRIPTION
**Pull request by viduxsh**
Added an Italian readme and added Italian files for the application language.

### CODE REVIEW

File: TwitchDownloaderWPF/Translations/Strings.it.resx [hunk 1]
```diff
@@ -0,0 +1,818 @@
...
+    <value>Frequenza di aggiornamento:</value>
+  </data>
+  <data name="UrlList" xml:space="preserve">
+    <value>Lista di URL</value>
+  </data>
+  <data name="VerboseErrorOutput" xml:space="preserve">
+    <value>Output di errore dettagliato</value>
+  </data>
+  <data name="VerboseErrors" xml:space="preserve">
+    <valueDettagli di errori:</value>
```
Comment (ScrubN) [inline TwitchDownloaderWPF/Translations/Strings.it.resx:556]: It seems the closing `>` got deleted accidentally

File: TwitchDownloaderWPF/Translations/Strings.it.resx [hunk 2]
```diff
@@ -0,0 +1,818 @@
...
+    <value>Accetta</value>
+  </data>
+  <data name="AddToQueue" xml:space="preserve">
+    <value>Aggiungi alla coda</value>
+  </data>
+  <data name="AlphaNotSupportedByCodec" xml:space="preserve">
+    <value>Hai selezionato un canale alfa (trasparenza) per un contenitore/codec che non lo supporta.\nRimuovi la trasparenza o codifica con MOV e RLE/PRORES (la dimensione del file sarà grande).</value>
+  </data>
+  <data name="ApplicationTheme" xml:space="preserve">
+    <value>Tema</value>
```
Comment (ScrubN) [inline TwitchDownloaderWPF/Translations/Strings.it.resx:130]: Missing trailing space

**Review by ScrubN — CHANGES_REQUESTED** [review]
This is awesome, but could you fix a few things?

**Comment by viduxsh** [comment]
I should have solved it, let me know.

**Review by ScrubN — APPROVED** [review]

**Comment by ScrubN** [comment]
Thanks for the additions!

**Pull request merged**